jerry-core/vm/vm-stack.c

```diff
@@ -75,7 +75,8 @@ vm_stack_context_abort (vm_frame_ctx_t *frame_ctx_p, /**< frame context */
         ecma_collection_chunk_t *chunk_p = MEM_CP_GET_NON_NULL_POINTER (ecma_collection_chunk_t,
                                                                         current);
 
-        ecma_free_value (*(ecma_value_t *) chunk_p->data, true);
+        lit_utf8_byte_t *data_ptr = chunk_p->data;
```

Comment:
I think this is an alignment problem, perhaps a double casting could help:

    ecma_free_value (*(ecma_value_t *) (void*) chunk_p->data, true);